### DISCUSSION TITLE
[Feature Request] [UI-related feature] Allow more fine-tuned control over pop-ups and pop-ins and "untargetable elements

### DISCUSSION THREAD

**Discussion by rubyFeedback** [uBlockOrigin/uBlock-issues#2815] [Q&A]
### Prerequisites

- [X] I verified that this is not a filter list issue. Report any issues with filter lists or broken website functionality in the [uAssets issue tracker](https://github.com/uBlockOrigin/uAssets/issues).
- [X] This is not a support issue or a question. For support, questions, or help, visit [/r/uBlockOrigin](https://www.reddit.com/r/uBlockOrigin/).
- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uBlock-issues/issues?q=is%3Aissue) to avoid opening a duplicate issue.
- [X] The issue is not present after disabling uBO in the browser.
- [X] I checked the [documentation](https://github.com/gorhill/uBlock/wiki) to understand that the issue I am reporting is not normal behavior.

### I tried to reproduce the issue when...

- [X] uBO is the only extension.
- [X] uBO uses default lists and settings.
- [X] using a new, unmodified browser profile.

### Description

So before I describe the thing here, let me state that there are WAY too many checkboxes here and what not. Most of these don't apply to my feature request; I was unable to find another entry, though, so the templates seem to restrict me. Anyway, now to the issue request.

I have been using ublock origin since many years and it is in general great.

Recently youtube ads do appear sometimes, in particular at the beginning of videos (oddly enough at the beginning, but not in the middle or elsewhere of a video), but this is not about ads from youtube here - I just wanted to mention that. Seems Google is fighting back. They used to run their "acceptable ads" propaganda tour in the past, then abandoned it. More recently they tried it with DRM-control of browsers. They really have an attitude problem, but this is an aside.

The issue here is what I see on old.reddit, but also other sites.

First, there is this red pop-up that sneaks in on the bottom left and then sneaks out. Aka some lame message notification nobody cares about from irrelevant dictators aka "mods". I'd love to be able to target such transient UI annoyances though. ublock origin seems to only target elements that are ON the very page, which makes sense for most users: elements rarely pop in and pop out on most websites. So the default is fine.

I'd love to be able to target that sneaky pop-in and other such crap. Google search also does this - they add a "other people also searched for RANDOM TOPIC I AM NOT INTERESTED IN". Aka a pop-in that emerges, and Google positions it right where I accidentally misclick. This happened to me several times, so this is 100% not an accident - Google WANTS to piss off users. Anyway. I never read the content on what is shown on that new tab that I accidentally click on, I just close it at once, but it would be nice if ublock origin could ALSO target such annoying UI elements that Google or reddit users.

So this is one part of the feature request - not sure whether it was filed before. Perhaps ublock origin could get an extended overview page or something like that, where we have more control over these things and which to block specifically. ESPECIALLY if these faded away already. Something like a "post mortem web page". I am not suggesting to make this the default. The default should be simple for people, fair enough. Non-tech savvy people use ublock origin. But I'd love something that allows us "power" users a bit more control. Perhaps it already exists, but I could not find it; in that case perhaps the UI element ublock origin offers could be extended. Nothing fancy, just something simple that allows us to target such sneaky UI choices others made to annoy us.

There is a second part to this suggestion. On kde-reddit, aka old-kde reddit, on the top left there is an irrelevant entry: "GET NEW REDDIT". I tried the new reddit and it SUCKS. It is useless for desktop users. So I want to block it.

In the past I was able to do so, but recently I can't target it. Whenever I choose "pick", some div-HTML tag overshadows this and I can't target the red thing anymore. I don't think this is so much ublock origin's fault and instead reddit being sneaky, but even then it would be kind of cool if ublock origin would allow some more finer-grained control, e. g. perhaps a "layer" next to pick or a "fine pick" option or anything that we can change to pick that red thing that annoys me.

In general ublock origin is by far the most important extension I use and it is awesome. So this is mostly a request for additional awesomeness, in particular against hostile, sneaky UI "choices" that pester, harass and abuse the user and visitor. I want to be in control of the browser, not vice versa (I am aware of the irony that I succumbed to the Google Empire by using Thorium, aka Google Chrome, but Firefox devs annoyed me a LOT with horrible UI choices and annoyances, so I abandoned that; I can't watch youtube video as I do not use pulseaudio, but via thorium it is just fine, and recompiling firefox via that horrible mozconfig file https://www.linuxfromscratch.org/blfs/view/svn/xsoft/firefox.html I am not willing to do - mozilla gave up on firefox years ago already.)

### A specific URL where the issue occurs.

https://old.reddit.com/r/kde/

### Steps to Reproduce

See above.

### Expected behavior

See above.

### Actual behavior

See above.

### uBO version

latest

### Browser name and version

thorium browser

### Operating System and version

linux, almost all of which I compiled from source

**Comment by gorhill** [comment]


**Reply by gorhill** [reply]
Converted to discussion, which is what appear OP is interested into.
